tardis/io/tests/test_HDFReaderWriter.py

```diff
@@ -0,0 +1,105 @@
+import os
+
+import numpy as np
+import pandas as pd
+import pandas.util.testing as pdt
+import pytest
+from astropy import units as u
+from astropy.tests.helper import assert_quantity_allclose
+from numpy.testing import assert_almost_equal, assert_array_almost_equal
+
+from tardis.io.util import HDFReaderWriter
+
+
+class Mock_HDF(HDFReaderWriter, object):
+    hdf_properties = ['property']
+    quantity_attrs = {}
+
+    def __init__(self, property):
+        self.property = property
+
+    def to_hdf(self, file_path, path=''):
+        buff_path = os.path.join(path, 'dummy')
```

Comment:
the path should be the path and a name (you use dummy, but we should use something else). set the `name` kwd to `None` and if it is `None` use the class name as the name